### DISCUSSION TITLE
ji-tang/info/Rhx064VmgucegLYCN9B8y

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#16] [Announcements]
# ji-tang/info/Rhx064VmgucegLYCN9B8y

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/ji-tang/info/Rhx064VmgucegLYCN9B8y?isInfo=false

**Comment by wurencaideli** [comment]
理发师：你的难看跟你的头发无关
笑死
